library/ssl_misc.h

```diff
@@ -2388,7 +2388,7 @@ static inline int mbedtls_ssl_tls13_sig_alg_for_cert_verify_is_supported(
     const uint16_t sig_alg)
 {
     switch (sig_alg) {
-#if defined(MBEDTLS_PK_CAN_ECDSA_SOME)
+#if defined(PSA_WANT_ALG_ECDSA)
```

Comment:
This needs to be `defined(PSA_WANT_ALG_ECDSA) || defined(PSA_WANT_ALG_DETERMINISTIC_ECDSA)`. IIRC we've already decided to add a definition for that in `include/psa/crypto_adjust_*.h`. I don't know if it's already being added by a PR in flight.

Reply:
`PSA_HAVE_ALG_SOME_ECDSA` from https://github.com/Mbed-TLS/mbedtls/pull/9384